app/controllers/editions_controller.rb

```diff
@@ -27,6 +30,25 @@ def show
   alias_method :metadata, :show
   alias_method :unpublish, :show
 
+  def update
+    params = permitted_params
+    @resource.assign_attributes(params[:edition])
+    success = @resource.update!
+
+    if success == true
```

Comment:
Don't think we really need the `success` variable.
```suggestion
    if @resource.update!
```